explorer/go.mod

```diff
@@ -0,0 +1,238 @@
+module explorer
```

Comment:
I believe we should use `github.com/ignite/plugins/explorer` instead of just `explorer`. This way would describe the real package URL.

Changing it would require to change the local imports too so they include the URL.

What do you think?

Reply:
Ah yes, I totally forgot about that. Thanks for mentioning!